Documentation/ceph-dashboard.md

```diff
@@ -72,6 +73,9 @@ The following dashboard configuration settings are supported:
   dashboard behind a proxy already served using SSL) by setting the `ssl` option
   to be false.
 
+* `role` Specifies a ceph dashboard role for admin user. For available roles
+  [see](https://docs.ceph.com/docs/master/mgr/dashboard/#user-roles-and-permissions)
```

Comment:
Is read-only actually a role for the dashboard? On this link it says:
```
read-only: provides read permission for all security scopes except the dashboard settings.
```
But in another example on that page it looks like the read-only role was applied to the dashboard, so perhaps that comment is outdated.

Reply:
My use case is to expose ceph dashboard and forbid admin to do management over it, just see current setting and status.

In my understanding is a role is for dashboard only.

So far PR reflects an easiest way to do so, as externally I'm struggling to catch time when dashboard got enabled by operator.